live-build/misc/ansible-roles/appliance-build.delphix-autofs/files/net.autofs

```diff
@@ -0,0 +1 @@
+/net	/etc/auto.net
```

Comment:
[The `devops-gate` version](https://gitlab.delphix.com/devops/devops-gate/blob/master/ansible/roles-shared/dlpx.common-ubuntu/tasks/autofs.yml) has `/net    -hosts`. I'm curious what the difference is between that and the version you have here. Should we change the `devops-gate` version to use `/net /etc/auto.net`?

Reply:
It looks like the `auto.net` version executes the shell script there to populate the map, and the `-hosts` version is built-in to the automount daemon. I'll switch this commit to use the `-hosts` version since that's the one that's documented in the `auto.master` man page.

Reply:
On the enabling of debug logging, if I recall, we did have an issue a few years back associated with auto-mounting home directories from pharos which was root-caused to a bug in the NFSv4 client on Illumos (not sure if it affected automounts on Linux images). Switching to NFSv3 mounts fixed that. I don't recall other issues that would have required client-side auto-mounter debugging. I'll leave this off for now unless we're actively chasing down some issue.